wallet/src/components/Proposal.tsx

```diff
@@ -259,8 +259,14 @@ const Proposal = ({ offer, purses, swingsetParams, beansOwing }) => {
       {offerArgs !== undefined && (
         <div className="OfferEntry">
           <h6>Arguments</h6>
-          <details style={{ whiteSpace: 'pre-wrap' }}>
-            {stringify(offerArgs, true)}
+          <details
+            style={{
+              whiteSpace: 'pre-wrap',
```

Comment:
why does this no longer fit on one line? the comma?

Reply:
I guess since I had started a new line the autoformat put a comma there instead of bringing it into one line, just fixed to be one line.